@here/harp-geoutils/lib/coordinates/GeoCoordinatesLike.ts

```diff
@@ -29,3 +29,33 @@ export function isGeoCoordinatesLike(object: any): object is GeoCoordinatesLike
         (typeof object.altitude === "number" || typeof object.altitude === "undefined")
     );
 }
+
+/**
+ * Wraps Coordinates around the antimeridian
+ *
+ * @param coords A List of GeoCoordinatesLike to wrap around the antimeridian
+ */
+export function wrapCoordinatesAround(coords: GeoCoordinatesLike[]) {
+    const antimerCrossIndex = coords.findIndex((val: GeoCoordinatesLike, index: number) => {
+        const prevLonIndex = index === 0 ? coords.length - 1 : index - 1;
+        const prevLon = coords[prevLonIndex].longitude;
+        const lon = val.longitude;
+
+        return prevLon > 90 && lon < -90;
+    });
+    if (antimerCrossIndex < 0) {
+        return;
+    }
+
+    for (let i = 0; i < coords.length; i++) {
+        const index = (antimerCrossIndex + i) % coords.length;
+        const currentLon = coords[index].longitude;
+        coords[index].longitude += 360;
+        const nextLon = coords[(index + 1) % coords.length].longitude;
+
+        if (currentLon < -90 && nextLon > 90) {
+            // new crossing in opposite direction, stop.
+            break;
+        }
+    }
+}
```

Comment:
This function is limited to polygons with sides spanning less than 180 degrees longitude.
It'd be simpler and more robust to just compute the GeoBox for the polygon to see where the southEast and northWest points are, then wrap around points with longitudes smaller than eastmost.
I can do that change in my PR, since here as I said above you don't need this function for GeoBox.

Reply:
ok